src/routes/+page.svelte

```diff
@@ -186,14 +186,103 @@
             </p>
         </div>
     </div>
-
+    
     <div class="text-center">
         <a class="inline-flex gap-x-4 items-center bg-green hover:bg-green-dark py-4 px-6 rounded-full text-xl font-ibm font-medium" href="https://eip2077.info">Explore EIP-2077 <span class="inline-block"><Explore /></span></a>
     </div>
+    
 </section>
 
-<section>
+<section class="flex flex-col gap-y-16 px-6 lg:px-20 py-16">
+    <div class="flex flex-col gap-y-4">
+        <h2 class="text-6xl font-inter font-medium">
+            Inde<span class="font-keania">x</span>
+        </h2>
+        <p class="font-inter text-2xl">
+            Recent work that has been put out by the collective.
+        </p>
+    </div>
+
+    <div class="flex flex-col gap-y-8 lg:flex-row lg:gap-x-8 lg:justify-center">
+        <div class="bg-black p-10 flex flex-col gap-y-6 flex-grow">
+            <p class="text-white text-xl font-inter font-medium">Website</p>
+            <img
+                src="/globe.webp"
+                alt="globe"
+                loading="lazy"
+                width="840"
+                height="840"
+            />
+            <p class="text-white text-xl font-inter font-medium max-w-screen-3xl">
+                See the geographical distribution of Ethereum validators in
+                real-time. A visual experience with in-depth metrics and
+                statistics.
+            </p>
+            <div>
+                <a
+                    class="inline-flex gap-x-4 items-center bg-green hover:bg-green-dark py-4 px-6 rounded-full text-xl font-ibm font-medium"
```

Comment:
In Figma, this link and all the others in this section are 1rem. `text-xl` is equivalent to `1.25rem`. Also font-family is inter not IBM
```suggestion
                    class="inline-flex gap-x-4 items-center bg-green hover:bg-green-dark py-4 px-6 rounded-full font-medium"
```